# Can't catch exception IndexError in Python?
File "/opt/django/fileupload/views.py" in upload_file
    56.         folder_info = url.split(':')[1]  &lt;-- This line
